### PR TITLE
[DEV-4483] Uplift user profile to new design

### PR DESCRIPTION
**Pull request by empanda**
https://roverdotcom.atlassian.net/browse/DEV-4483

### CODE REVIEW

**Comment by empanda** [comment]
@kevinvanderlugt please review and +1. Thanks.

File: askbot/static/askbot/css/user_profile.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            margin-left: 35px;
+        }
+
+        .user-info-table {
+            .avatar {
+                margin: 0 15px 0 0;
+
+                .avatar-box .gravatar {
+                    border: none;
+                    border-radius: 0;
```
Comment (kevinvanderlugt) [inline askbot/static/askbot/css/user_profile.scss]: I talked with CJ and she said that she made a mistake and the profile image should have a radius.

File: askbot/templates/user_profile/user_stats.html [hunk 2]
```diff
@@ -7,54 +7,54 @@
...
+    {% if total_votes %}
+        <a name="votes"></a>
+        <h3>{{ total_votes }} vote{{ total_votes|pluralize }}</h3>
+        <div class="user-stats-table">
+            <div class="users-questions">
+                <div class="counts">
+                    <div class="votes some-votes">
+                        <span class="item-count">{{ up_votes }}</span>
+                    </div>
+                    <div class="votes-down some-votes">
```
Comment (kevinvanderlugt) [inline askbot/templates/user_profile/user_stats.html]: This section appears as a thumbs up but should be a thumbs down.

File: askbot/templates/user_profile/user_network.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
 {% block usercontent %}
```
Comment (kevinvanderlugt) [inline askbot/templates/user_profile/user_network.html]: Should this section get the .user-profile-page body tag?  It seems like this is the only profile page tab that has the tab bar below the users name

File: askbot/templates/user_profile/user_stats.html (hunk 2 above)
Comment (empanda) [inline askbot/templates/user_profile/user_stats.html]: I'll fix it up. It is waiting on https://github.com/roverdotcom/web/pull/8541.

**Comment by empanda** [comment]
@kevinvanderlugt ready for you again.

**Comment by kevinvanderlugt** [comment]
Oops commented on the commit heh.
:+1:

**Pull request merged**